docs/commands/graph.query.md

```diff
@@ -926,6 +926,7 @@ This section contains information on all supported functions from the Cypher que
 | [reduce(...)](#reduce)               | Returns a scalar produced by evaluating an expression against each list member                                                                                                                       |
 | list.union(_expr_)                         | Given two lists, return their union <br> Returns null when both v1 and v2 are null. <br> When list is evaluated to null - it is first replaced with an empty list <br> If dupPolicy is evaluated to 0 (set semantics): <br> If a value appears x>0 times in v1 or y>0 times in v2 it will appear one time in the result. <br> The result is ordered ascendingly. <br> If dupPolicy is evaluated to 1 (bag semantics): <br> If a value appears x times in v1 and y times in v2 - it will appear x+y times in the result. <br> Lists are concatenated without reordering. <br> If dupPolicy is evaluated to 2 (bag semantics, alternative definition): <br> If a value appears x times in v1 and y times in v2 - it will appear max(x,y) times in the result. <br> Ordering: lists are concatenated with duplicates removed from the end of the list. |
 | list.intersection(_expr_)                         | Given two lists, return their intersection <br> Returns null when both v1 and v2 are null. <br> When v1 is evaluated to null - it is first replaced with an empty list <br> When v2 is evaluated to null - it is first replaced with an empty list <br> If dupPolicy is evaluated to 0 (set semantics): <br> If a value appears x>0 times in v1 and y>0 times in v2 it will appear one time in the result. <br> The result is ordered ascendingly. <br> If dupPolicy is evaluated to 1 (bag semantics): <br> If a value appears x times in v1 and y times in v2 it will appear min(x, y) times in the result. <br> The ordering of the elements in the result is similar to that of v1 with duplicates removed from the end of the list. |
+| list.diff(_expr_)                         | Given two lists, return their difference (v1-v2). <br> Returns null when v1 is null. <br> When v2 is evaluated to null - it is first replaced with an empty list <br> If dupPolicy is evaluated to 0 (set semantics): <br> If a value appears at least once in v2 - it will not appear in the result. <br> If a value appears at least once in v1 and does not appear in v2 - it will appear once in the result. <br> The result is ordered ascendingly. <br> If dupPolicy is evaluated to 1 (bag semantics): <br> If a value appears x times in v1 and y times in v2 - it will appear max(x-y, 0) times in the result. <br> The ordering of the elements in the result is similar to that of v1 with duplicates removed from the end of the list. <br> If dupPolicy is evaluated to 2 (remove all instances): <br> If a value appears at least once in v2 - it will not appear in the result. <br> If a value appears x times in v1 and does not appear in v2 - it will appear x times in the result. <br> The ordering of the elements in the result is similar to that of v1. |
```

Comment:
Please update the function's signature.
Also and `(Since RedisGraph v2.12)` as the first line of the description.